cmd/clusterctl/client/cluster/ownergraph.go

```diff
@@ -65,7 +65,7 @@ func GetOwnerGraph(ctx context.Context, namespace, kubeconfigPath string) (Owner
 }
 
 func discoverOwnerGraph(ctx context.Context, namespace string, o *objectGraph) (OwnerGraph, error) {
-	selectors := []client.ListOption{}
+	var selectors []client.ListOption
```

Comment:
I would advice against making changes like this in general.

Either we get a consensus on which variant we prefer or we shouldn't change this back and forth based on what is then basically personal preference

Reply:
I agree, we should go for a standard implementation for all such use case. An empty slice is the preferred way according to the [go wiki ](https://github.com/golang/go/wiki/CodeReviewComments#declaring-empty-slices)

Reply:
Ah perfect. CodeReviewComments as a reference is pretty good. Let's go with this option than

(I didn't know/remember that they covered this case)